test/Gemfile.lock

```diff
@@ -20,4 +20,4 @@ DEPENDENCIES
   test-unit
 
 BUNDLED WITH
-   1.16.4
+   2.1.4
```

Comment:
Maybe revert this bit and see if it fixes the build magically?

Reply:
`RUN gem install bundler -v 1.17.3 --no-ri --no-rdoc` is in most/all of the dockerfiles. I'll manually set the `BUNDLED WITH` to `1.17.3`.